assignment04/js/myScript4.js

```diff
@@ -0,0 +1,52 @@
+$(document).ready(function() {
+    $('#action').click(function() {
+
+        var letterGrade = $('#letter-grade').val().toUpperCase();
+        
+        var myFavoriteThings = [];
+            myFavoriteThings.push("Days Off ");   
+            myFavoriteThings.push("Sleeping "); 
+            myFavoriteThings.push("Streaming Movies / TV "); 
+            myFavoriteThings.push("No Phone Calls "); 
+
+        //Write your decision structure between this comment...
+        
+        if (letterGrade == "A"){
+            $('#decision-holder').text("Grade range for A: 90% to 100%");
+        } 
+        else if (letterGrade == "B"){
+            $('#decision-holder').text("Grade range for B: 80% to 90%");
+        }
+        else if (letterGrade == "C"){
+            $('#decision-holder').text("Grade range for C: 70% to 80%");
+        }
+        else if (letterGrade == "D"){
+            $('#decision-holder').text("Grade range for D: 60% to 70%");
+        }
+        else if (letterGrade == "F"){
+            $('#decision-holder').text("Grade range for F: Less than 60%");
+        }
+        else {
+            $('#decision-holder').text("Error: invalid value input!");
+        };

```

Comment:
You don't need a semicolon after that curly brace. Otherwise - nice if/else 👍

Reply:
Okay, thanks. I was wondering about that, and just decided to be safe than sorry. I'll leave out the semicolons after brackets from now on.